_config.yml

```diff
@@ -1,2 +1,4 @@
 gems:
   - jekyll-redirect-from
+
+theme: jekyll-theme-cayman
```

Comment:
We are using `jekyll-redirect-from` only for the redirects. What is the usage of the `jekyll-theme-cayman` theme?